configs/components/puppet.json

```diff
@@ -1 +1 @@
-{"url":"git@github.com:puppetlabs/puppet.git","ref":"refs/tags/7.29.0"}
+{"url":"git@github.com:puppetlabs/puppet.git","ref":"090122aae2390393762d0bc335ed7047c4bdb8a5"}
```

Comment:
I think this needs to be https://github.com/puppetlabs/puppet/commit/565cf369602465df9e1526f34debee17b3c8a0d3